src/components/readJsonFile.java

```diff
@@ -0,0 +1,109 @@
+//this code read data from universalgamemaker-export.json,
```

Comment:
Two last things:
- can you move this file into `scripts` folder
- Instead of hard coding your local paths in script use relative paths.